### PR TITLE
Add functions of student-dashboard

### LINKED ISSUE #72: Calculate the amount of reviews a student has left
Create a function that calculates the amount of reviews a student has left

reviews
teachers
students

### PR DESCRIPTION
**Pull request by christopher1030123**
Fixes #72 .

Changes made in this pull request:
Create Student-dashboard Service which includes three functions:
1.output the reviews a student has completed
2.output the reviews a student has not completed
3.output the percentage of reviews a student completed

### CODE REVIEW

**Comment by christopher1030123** [comment]
The StudentDashBoard Service is changed to Student Service

Unit Test is also added

File: src/app/student/student.service.ts [hunk 1]
```diff
@@ -4,14 +4,18 @@ import { Headers, Http } from '@angular/http';
 import 'rxjs/add/operator/toPromise';
 
 import { Student } from './student';
+import{Review} from '../review/review';
```
Comment (ltcarbonell) [inline src/app/student/student.service.ts]: `import { Review }`

File: src/app/student/student.service.ts [hunk 2]
```diff
@@ -4,14 +4,18 @@ import { Headers, Http } from '@angular/http';
...
+import {Course} from '../course/course';
```
Comment (ltcarbonell) [inline src/app/student/student.service.ts]: `import { Course }`

File: src/app/student/student.service.ts [hunk 3]
```diff
@@ -4,14 +4,18 @@ import { Headers, Http } from '@angular/http';
...
 @Injectable()
 export class StudentService {
 
     result: any;
     private headers = new Headers({ 'Content-Type': 'application/json' });
     private studentsUrl = 'api/students';  // URL to students api
-
+    courses : Course[] = [];
+    Notreviewed: Course[]=[];
+    Reviews:Review[]=[];
```
Comment (ltcarbonell) [inline src/app/student/student.service.ts]:
```
Notreviewed: Course[] = [];
Reviews:Review[] = [];
```

File: src/app/student/student.service.ts [hunk 4]
```diff
@@ -92,6 +96,42 @@ export class StudentService {
...
+
+  // getPercentageofReviews
+  getPercentageReview(id: string): number {
+    this.getCoursesByStudent(id)
+      .then(course=>this.courses=course);
+    this.getReviewsByStudent(id)
+      .then(Res=>this.Reviews=Res);
+    let percent:number=this.Reviews.length/this.courses.length;
+    return percent;
+
```
Comment (ltcarbonell) [inline src/app/student/student.service.ts]: extra new line

**Comment by ltcarbonell** [comment]
Merging

**Pull request merged**